src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -1520,17 +1530,20 @@ pre.rust {
 #settings-menu, #help-button {
 	margin-left: 4px;
 	outline: none;
+	background-color: var(--button-background-color);
 }
 
 #copy-path {
 	height: 34px;
+	background-color: var(--main-background-color);
 }
 #settings-menu > a, #help-button > button, #copy-path {
 	padding: 5px;
 	width: 33px;
 	border: 1px solid;
 	border-radius: 2px;
 	cursor: pointer;
+	border-color: var(--border-color);
```

Comment:
This should be combined with the `border` property on line 1543 above.

Reply:
I'm quite ashamed of this one...